backend-project/small_eod/notifications/views.py

```diff
@@ -0,0 +1,74 @@
+from django.forms.models import model_to_dict
+from rest_framework.views import APIView
+
+
+class SendNotificationsMixin(APIView):
```

Comment:
(Pytanie do @ad-m, ponieważ nie jestem w 100% przekonany czy mam rację.)

Czy nasz Mixin powinien dziedziczyć po pełnoprawnej klasie (tj. nie po innym Mixinie), czy powinniśmy pozwolić klasie używającej naszego mixinu na dodanie APIView? W tym momencie nasz `SendNotificationMixin` nie jest sensu stricto Mixinem, a pełnoprawną klasą, o ile dobrze kojarzę.

Reply:
A no nie jest mixinem, trudno będzie nie dziedziczyć po ApiView, więc można po prostu zmienić nazwę tej klasy na `SendNotificationsView` lub `NotificationsView`

Reply:
Moim zdaniem należaloby użyć `class SendNotificationsMixin`. Wówczas powiadomienia są tylko czymś dodawanym do jakiegoś istniejącego widoku, a nie definiują kolejności klas w żadnym widoku. Wydaje mi się, że to wystarczy, aby uznać tę klasę za Mixin.

Oczywiście, z drugiej strony należy zadbać o kolejność klasy, więc zapisać raczej jako:

```
class CaseViewSet(NotificationsMixin, viewsets.ModelViewSet):
```